codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/testutil/Rust.kt

```diff
@@ -177,6 +179,10 @@ fun TestWriterDelegator.compileAndTest(runClippy: Boolean = false) {
         manifestCustomizations = emptyMap(),
         libRsCustomizations = listOf(),
     )
+    println("Generated files:")
+    generatedFiles().forEach { path ->
```

Comment:
Is this already sorted by name?

Reply:
no it's actually nondeterministic 😅 probably nice for usability to sort it